utils/hct/hctdb.py

```diff
@@ -2417,6 +2417,7 @@ def build_valrules(self):
         self.add_valrule("Sm.InvalidTextureKindOnUAV", "Texture2DMS[Array] or TextureCube[Array] resources are not supported with UAVs")
         self.add_valrule("Sm.InvalidResourceKind", "Invalid resources kind")
         self.add_valrule("Sm.InvalidResourceCompType","Invalid resource return type")
+        self.add_valrule("Sm.InvalidSamplerFeedbackType","Invalid sampler feedback type")
```

Comment:
TBD: Need to add rule for "texture resource provided must be the same for all operations on the same feedback resource", or something like that.